install/0000_30_machine-api-operator_15_ironic-exporter-prometheusrules.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: metal3-baremetalhost-controller
+  namespace: openshift-machine-api
+spec:
+  groups:
+  - name: metal3-baremetalhost-controller
+    rules:
+    - alert: HighCPUTemperature
```

Comment:
Are those real alerts?

Reply:
I've created these alerts based on the values from the Hardware specification that we are going to use. I will send you the doc

Reply:
Ack